DSharpPlus/Entities/DiscordAuditLogObjects.cs

```diff
@@ -23,902 +23,1051 @@
 
 using System;
 using System.Collections.Generic;
+using DSharpPlus.Enums;
 
-namespace DSharpPlus.Entities
+namespace DSharpPlus.Entities;
```

Comment:
as a principal note, i really dislike how this file contains so many different types, but it's not the end of the world

in an ideal world we'd just sourcegen all this

Reply:
I could move them to own files if its prefered

Reply:
would be, but it's not the end of the world if we keep it here